diagnostics-extension/src/dpsl.types.ts

```diff
@@ -236,3 +236,32 @@ export interface DiskReadRoutineParams {
   lengthSeconds: number;
   fileSizeMB: number;
 }
+
+export type DpslTypes =
```

Comment:
Needed for tests and might be useful in the app as well.

Reply:
You just needed TelemetryInfo in the tests. You don't need *Params here. Please remove them.

Reply:
and you don't need inner objects:

example: you need CpuInfo but not inner objects like: CpuCStateInfo, LogicalCpuInfo, PhysicalCpuInfo.

Same for the other types.